src/tensorflow.cc

```diff
@@ -1295,7 +1295,7 @@ AutoCompleteHelper::FixBatchingSupport()
     if (model_support_batching_ == 1) {
       LOG_MESSAGE(
           TRITONSERVER_LOG_WARN,
-          (std::string("autofilled max_batch_size to 1 for model '") +
+          (std::string("autofilled max_batch_size to 4 for model '") +
            model_state_->Name() +
            "' since batching is supporrted but no max_batch_size is specified "
            "in model configuration. Must specify max_batch_size to utilize "
```

Comment:
Must specify max_batch_size manually to use a different max batch size than the default.